src/components/Button/Button.module.css

```diff
@@ -16,20 +16,22 @@
 
 /**
  * Primary disabled button styles
+ * 1) Class is doubled to increase specificity
```

Comment:
This is now required because the `ClickableStyle` is pairing the variant and status classes, which bumped its specificity.